test-crawler/config_test.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v2"
+)
+
+func TestNewConfig(t *testing.T) {
+	cfg := Config{
+		TestCrawlerPaths:      []string{"a", "b"},
+		TestCrawlerOutputMode: "stdout",
+		TestCrawlerOutputDir:  "output",
+		TestCrawlerLanguage:   "go",
+		TestCrawlerIndentJSON: true,
+		TestCrawlerIgnore:     []string{"extern"},
+
+		BehaviorGenPaths:     []string{"c", "d"},
+		BehaviorGenIgnore:    []string{".git"},
+		BehaviorGenOutputDir: "output/catalogue",
+	}
+
+	f, err := ioutil.TempFile("", "config-test.yml")
```

Comment:
Why not `t.TempFile` ?

Reply:
from some reason there is no `t.TempFile`, only `t.TempDir`